File: scripts/capture_web_probe.py
from playwright.sync_api import sync_playwright


with sync_playwright() as p:
    browser = p.chromium.launch(headless=True)
    page = browser.new_page(viewport={"width": 1440, "height": 1000}, device_scale_factor=1)
    page.goto("http://127.0.0.1:8082/", wait_until="domcontentloaded", timeout=60000)
    page.wait_for_timeout(5000)
    print(f"url={page.url}")
    print(f"title={page.title()}")
    print(page.locator("body").inner_text()[:3000])
    page.screenshot(path="E:/Grana-temporarios/prints/web-probe.png", full_page=True)
    browser.close()

File: scripts/capture_web_probe.mjs
import { chromium } from 'playwright';

const browser = await chromium.launch({ headless: true });
const context = await browser.newContext({ viewport: { width: 1440, height: 1000 }, deviceScaleFactor: 1 });
const page = await context.newPage();
await page.goto('http://127.0.0.1:8082/', { waitUntil: 'domcontentloaded', timeout: 60000 });
await page.waitForTimeout(5000);
console.log(`url=${page.url()}`);
console.log(`title=${await page.title()}`);
console.log((await page.locator('body').innerText()).slice(0, 3000));
await page.screenshot({ path: 'E:/Grana-temporarios/prints/web-probe.png', fullPage: true });
await browser.close();
